lib/main.dart

```diff
@@ -105,3 +109,86 @@ class UserButton extends StatelessWidget {
     );
   }
 }
+
+class SurveyButton extends StatelessWidget {
+  const SurveyButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 20),
+      child: FilledButton(
+        onPressed: () => navigator.push(const SurveyPicker()),
```

Comment:
I added another button to the `ChooseAnyView` screen that shows a few sample surveys.

(Eventually these surveys will be integrated with different parts of the app!)